tensorflow/tensorflow_eager_simple.py

```diff
@@ -63,7 +63,7 @@ def create_optimizer(trial):
         kwargs["learning_rate"] = trial.suggest_float(
             "rmsprop_learning_rate", 1e-5, 1e-1, log=True
         )
-        kwargs["decay"] = trial.suggest_float("rmsprop_decay", 0.85, 0.99)
+        kwargs["weight_decay"] = trial.suggest_float("rmsprop_weight_decay", 0.85, 0.99)
```

Comment:
In the description of this PR, you mentioned that this example users `tf.keras.optimizers`, but it tries to get the optimizers from `tf.optimizers` in L76. Do you think we can replace it with `tf.keras.optimizers`?

https://github.com/nzw0301/optuna-examples/blob/24f61956f9f904a8877d9d54404f73904f27f256/tensorflow/tensorflow_eager_simple.py#L76

Reply:
Thank you for your confirmation. Then, we can keep `tf.optimizer` here.